scene/multiplayer/multiplayer_spawner.cpp

```diff
@@ -97,12 +98,21 @@ void MultiplayerSpawner::add_spawnable_scene(const String &p_path) {
 	}
 	spawnable_scenes.push_back(sc);
 }
+
 int MultiplayerSpawner::get_spawnable_scene_count() const {
+	if (spawnable_scenes.is_empty()) {
+		return 0;
+	}
 	return spawnable_scenes.size();
 }
```

Comment:
Looking at the source code of LocalVector, I don't see why this is needed. `is_empty()` (inline) checks if `count == 0`, while `size()` (inline) returns `count`.

Reply:
It crashes though. I don't know why.

Reply:
I can't reproduce the crash at all with `get_spawnable_scene_count()` (Arch on 5.15.41-1-lts). I can reproduce the crash with the other method though. See my comment there.